packages/react-native-web/src/modules/ImageLoader/index.js

```diff
@@ -111,37 +119,78 @@ const ImageLoader = {
       if (typeof failure === 'function') {
         failure();
       }
-      ImageLoader.abort(requestId);
+      ImageLoader.release(requestId);
       clearInterval(interval);
     }
   },
-  has(uri: string): boolean {
+  has(uri: ?string): boolean {
     return ImageUriCache.has(uri);
   },
-  load(uri: string, onLoad: Function, onError: Function): number {
+  load(
+    source: ImageSource,
+    onLoad: (ImageResult) => void,
+    onError: Function
+  ): number {
     id += 1;
     const image = new window.Image();
-    image.onerror = onError;
-    image.onload = (e) => {
+
+    const handleLoad = () => {
       // avoid blocking the main thread
-      const onDecode = () => onLoad({ nativeEvent: e });
-      if (typeof image.decode === 'function') {
-        // Safari currently throws exceptions when decoding svgs.
-        // We want to catch that error and allow the load handler
-        // to be forwarded to the onLoad handler in this case
-        image.decode().then(onDecode, onDecode);
-      } else {
-        setTimeout(onDecode, 0);
-      }
```

Comment:
Removed the `if (typeof image.decode === 'function')` check and the `setTimeout` fallback, because the minimal supported browser versions are all capable of `image.decode`